helm-charts/common/tgi/README.md

```diff
@@ -4,26 +4,14 @@ Helm chart for deploying Hugging Face Text Generation Inference service.
 
 ## Installing the Chart
 
-To install the chart, run the following:
+Before installation, you need to export `HFTOKEN`.
 
 ```console
-cd GenAIInfra/helm-charts/common
-export MODELDIR=/mnt/opea-models
-export MODELNAME="bigscience/bloom-560m"
 export HFTOKEN="insert-your-huggingface-token-here"
-helm install tgi tgi --set global.modelUseHostPath=${MODELDIR} --set LLM_MODEL_ID=${MODELNAME} --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN}
-# To deploy on Gaudi enabled kubernetes cluster
-# helm install tgi tgi --set global.modelUseHostPath=${MODELDIR} --set LLM_MODEL_ID=${MODELNAME} --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} --values gaudi-values.yaml
+helm repo add opea https://opea-project.github.io/GenAIInfra
```

Comment:
As there are discussions about where to release the helm charts, I think it would be better to put only chart options and verification part in the README.
The deployment part might be different if users are getting the helm chart from various places.
Let's involve @poussa @eero-t to bring more experience. (For a better readme before v1.1 release)

Reply:
The less duplication, the better.  There could e.g. be a link to project README instead:
https://github.com/opea-project/GenAIInfra/blob/main/helm-charts/README.md#deploy-with-helm-charts